Always check stress in is_converged() if present in results

ikowalec opened the pull request:
Formerly, if `'forces'` and `'stress'` were present, `'forces'` would take priority and `'stress'` was not checked. 

Now both properties are checked against `fmax` if available in `Atoms.calc.results`

ikowalec committed:
Always check stress if present in calc.results

logsdail committed:
Merge branch 'main' into forces_bugfix

logsdail commented:
There seems to be a new weird error relating to the requirements file in the CI, which I don’t think you’ve touched?

ikowalec commented:
> There seems to be a new weird error relating to the requirements file in the CI, which I don’t think you’ve touched?

No, I have not. The tests are passing locally.

logsdail committed:
Merge branch 'main' into forces_bugfix

logsdail reviewed (approved):
This seems all self-explanatory and so if the tests pass I'll merge.

GaryLZW committed:
Merge branch 'main' into forces_bugfix

logsdail committed:
Change phonopy installation to version 3.2.0

Revert phonopy installation to version 3.2.0 due to compatibility issues with the new version.

logsdail committed:
Modify phonopy installation version constraint

Update phonopy installation to use version less than 4.0.

logsdail committed:
Change phonopy installation to version <4

Updated phonopy installation constraint to version 4.

logsdail committed:
Fix syntax for installing phonopy in workflow

logsdail committed:
Merge branch 'main' into forces_bugfix
